.github/workflows/vk-gl-cts-nightly.yml

```diff
@@ -0,0 +1,56 @@
+name: VK-GL-CTS Nightly
+
+on:
+  schedule:
+    - cron: '00 07 * * *'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+env:
+  DISABLE_CTS_SLANG: 0
+jobs:
+  build:
+    runs-on: [Windows, self-hosted]
+    timeout-minutes: 100
+    continue-on-error: true
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        submodules: 'true'
+        fetch-depth: '0'
+    - name: setup-msbuild
+      uses: microsoft/setup-msbuild@v1
+    - name: build slang
+      run: |
+        .\premake.bat vs2019 --arch=x64 --deps=true --no-progress=true --enable-cuda=true
+        
+        .\make-slang-tag-version.bat
+        
+        MSBuild.exe slang.sln -v:m -m -property:Configuration=Release -property:Platform=x64 -property:WindowsTargetPlatformVersion=10.0.19041.0 -maxcpucount:12
+    - uses: robinraju/release-downloader@v1.7
+      with:
+        latest: true
+        repository: "shader-slang/VK-GL-CTS"
+        fileName: "VK-GL-CTS_WithSlang-0.0.3-win64.zip"
```

Comment:
The filename shouldn't include a version number of it since it will never be accurate?